music/checkpoints/README.md

```diff
@@ -56,7 +56,9 @@ multitrack_med_fb256|MusicVAE|A larger 1-bar multitrack model, trained with 256
 multitrack_chords|MusicVAE|A 1-bar chord-conditioned multitrack model, trained with 64 free bits. Quantized to 1-byte weights.|26.9|[Right Click to Copy](https://storage.googleapis.com/magentadata/js/checkpoints/music_vae/multitrack_chords)
 multitrack_med_chords|MusicVAE|A larger 1-bar chord-conditioned multitrack model, trained with 64 free bits. Quantized to 1-byte weights.|96.9|[Right Click to Copy](https://storage.googleapis.com/magentadata/js/checkpoints/music_vae/multitrack_med_chords)
 multitrack_med_chords_fb256|MusicVAE|A larger 1-bar chord-conditioned multitrack model, trained with 256 free bits. Quantized to 1-byte weights.|96.9|[Right Click to Copy](https://storage.googleapis.com/magentadata/js/checkpoints/music_vae/multitrack_med_chords_fb256)
-groovae_unquantize_4bar|MusicVAE|A 4-bar GrooVAE model, trained to add feel to 9-class quantized drum beats.|15.8|[Right Click to Copy](https://storage.googleapis.com/magentadata/js/checkpoints/music_vae/groovae_unquantize_4bar)
+groovae_2bar_humanize|MusicVAE|A 2-bar GrooVAE model that converts a quantized, constant-velocity drum pattern into a 'humanized' groove.|15.8|[Right Click to Copy](https://storage.googleapis.com/magentadata/js/checkpoints/music_vae/groovae_2bar_humanize)
+groovae_2bar_tap2drum|MusicVAE|A 2-bar GrooVAE model that converts a constant-velocity single-drum 'tap' pattern into a groove.|15.8|[Right Click to Copy](https://storage.googleapis.com/magentadata/js/checkpoints/music_vae/groovae_2bar_tap2drum)
+groovae_4bar|MusicVAE|A 4-bar GrooVAE autoencoder.|15.8|[Right Click to Copy](https://storage.googleapis.com/magentadata/js/checkpoints/music_vae/groovae_4bar)
```

Comment:
is 4bar like 2bar_humanize but 4 bars? the naming seems weird here, maybe.

Reply:
no, it is an autoencoder (same input and output) more akin to normal MusicVAE models